src/diffusers/models/vae_flax.py

```diff
@@ -87,6 +109,22 @@ def __call__(self, hidden_states):
 
 
 class FlaxResnetBlock2D(nn.Module):
+    """
+    Flax implementation of 2D Resnet Block.
+
+    Args:
+        in_channels (`int`):
+            Input channels
+        out_channels (`int`):
+            Output channels
+        dropout_prob (:obj:`float`, *optional*, defaults to 0.0):
```

Comment:
```suggestion
        dropout (:obj:`float`, *optional*, defaults to 0.0):
```
#595 being merged we can replace it